### ISSUE TITLE
Stop using or encouraging mktap in the documentation

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3247]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)| @radix reported|
|-|-|
|Trac ID|trac#3247|
|Type|enhancement|
|Created|2008-05-20 23:54:40Z|

It's been seriously deprecated at this point, and we still have a number of references to it in the documentation (references that don't clearly indicate that it's deprecated).

<details><summary>Searchable metadata</summary>

```
trac-id__3247 3247
type__enhancement enhancement
reporter__radix radix
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__duplicate duplicate
component__core core
keywords__ 
time__1211327680000000 1211327680000000
changetime__1226266632000000 1226266632000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_1"></a>|@thijstriemstra set status to `closed`|
|-|-|

Duplicate of fixed [#3393](https://github.com/twisted/twisted/issues/3393)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
